src/main/kotlin/no/nav/syfo/personoppgavehendelse/PublishPersonoppgavehendelseService.kt

```diff
@@ -1,15 +1,48 @@
 package no.nav.syfo.personoppgavehendelse
 
-import no.nav.syfo.database.DatabaseInterface
-import no.nav.syfo.personoppgave.domain.PersonOppgave
+import no.nav.syfo.personoppgave.*
+import no.nav.syfo.personoppgave.domain.*
+import org.slf4j.LoggerFactory
+import java.sql.Connection
 
 class PublishPersonoppgavehendelseService(
-    private val database: DatabaseInterface,
+    private val connection: Connection,
     private val personoppgavehendelseProducer: PersonoppgavehendelseProducer,
 ) {
+    private val log = LoggerFactory.getLogger(PublishPersonoppgavehendelseService::class.java)
+
     fun getUnpublishedOppgaver(): List<PersonOppgave> {
-        return emptyList()
+        return connection.getPersonOppgaverByPublish(publish = true).toPersonOppgaver()
```

Comment:
Dette blir dumt siden dere ikke får lukket connection. Så bedre med database for da kan dere gjøre

database.connection.use {...}